speech/google/cloud/speech/sample.py

```diff
@@ -69,6 +71,15 @@ def __init__(self, content=None, source_uri=None,
             raise ValueError('Invalid encoding: %s' % (encoding,))
 
     @property
+    def chunk_size(self):
+        """Chunk size to send over GRPC. ~100ms
+
+        :rtype: int
+        :returns: Optimized chunk size.
+        """
+        return int(self.sample_rate / 10)
```

Comment:
@tseaver in python 2, if they happen to pass in a sample rate of say...16000.0, then `16000.0//10` returns `1600.0` which then fails when you call `read(1600.0)`.

`TypeError: 'float' object cannot be interpreted as an index`.
A rare case I suppose.